tests/rptest/tests/cluster_quota_test.py

```diff
@@ -405,11 +405,12 @@ def test_client_response_throttle_mechanism_applies_to_next_request(self):
             bootstrap_servers=self.leader_node,
             client_id="consumer",
             consumer_timeout_ms=1000,
-            max_partition_fetch_bytes=self.target_default_quota_byte_rate * 10,
+            max_partition_fetch_bytes=self.break_default_quota_message_amount *
+            self.message_size,
```

Comment:
Can you help me understand this change?

Reply:
Added more context into the commit message now: https://github.com/redpanda-data/redpanda/pull/18098/commits/096f273b1ea3bf6c689eaaccbd1655f99d19400c

Originally this change was required because it also increased the data fetched from 10x to 11x the quota rate. If we allow bursting and have a 10x burst rate configured we need this because the token bucket gets refilled between the 2 fetches so we need to have the token bucket go enough into the negative to not replenish by the next fetch. This is now not required if we don’t allow bursting, but I think the change is worth keeping for the other 2 points shown in the commit message.